src/torchmetrics/functional/classification/precision_recall_curve.py

```diff
@@ -14,6 +14,7 @@
 
 from typing import List, Optional, Sequence, Tuple, Union
 
+import humanfriendly
```

Comment:
I saw that one coming haha.

So, i put it anyway because it feels like the kind of functionality pruned to silly mistakes while a tiny library like this has it neatly packed in.

I can try to make a minimal version of it based on the library's source code. Is that a better solution?

Reply:
lets make it conditional, if user already have it, then use it
```py
if module_available("humanfriendly"):
    import humanfriendly
else:
    humanfriendly = None
```